.github/workflows/ci.yml

```
@@ -37,7 +37,7 @@ jobs:
           # avoid building unrelated products for testing by specifying the test product explicitly
           swift build --product TokamakPackageTests
           `xcrun --find xctest` .build/debug/TokamakPackageTests.xctest ||
-            find /var/folders -iname SnapshotTests -exec cp -r {} . \;
+            cp -r `find /var/folders -iname RenderingTests` . && exit 1
```

This doesn't work as intended. See [this](https://unix.stackexchange.com/questions/88850/precedence-of-the-shell-logical-operators).

```suggestion
            (cp -r `find /var/folders -iname RenderingTests` . ; exit 1)
```